pypeit/data/utils.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+"""
+Data locations for built-in PypeIt data files
+
+.. include:: ../include/links.rst
+"""
+import os
+from pkg_resources import resource_filename
+
+from linetools.spectra import xspectrum1d
+
+from pypeit import io
+from pypeit import msgs
+
+
+class Paths():
+    """Paths List of hardwired paths within the pypeit.data module
+
+    [extended_summary]
+    """
+
+    # Class Attributes -- Hardwired Paths
+    data = resource_filename('pypeit', 'data')
+
+    # Telluric Corrections
+    telgrid = os.path.join(data, 'telluric', 'atm_grids')
+    tel_model = os.path.join(data, 'telluric', 'models')
+
+    # Wavelength Calibrations
+    arclines = os.path.join(data, 'arc_lines')
+    reid_arxiv = os.path.join(data, 'arc_lines', 'reid_arxiv')
+    linelist = os.path.join(data, 'arc_lines', 'lists')
+    nist = os.path.join(data, 'arc_lines', 'NIST')
+    arc_plot = os.path.join(data, 'arc_lines', 'plots')
+
+    # Flux Calibrations
+    standards = os.path.join(data, 'standards')
+    extinction = os.path.join(data, 'extinction')
+    skisim = os.path.join(data, 'skisim')
+    filters = os.path.join(data, 'filters')
+    sensfuncs = os.path.join(data, 'sensfuncs')
+
+    # Other
+    sky_spec = os.path.join(data, 'sky_spec')
+    static_calibs = os.path.join(data, 'static_calibs')
+
+    def __init__(self):
+        pass
+
+
+
+# Loading Functions for Particular File Types ================================#
+def load_telluric_grid(filename):
```

Comment:
I'm curious what others think of consolidating these load functions here.  I think it's a step forward, but I'm interested in other opinions.  It's worth noting that we have another module that's supposed to do a lot of file loading, `pypeit/core/load.py`, but I don't know how much it's actually used anymore...

Reply:
I hadn't stumbled across `pypeit/core/load.py`, and would be very happy to place load functions there.  It felt useful to consolidate the load functions I did come across -- final disposition is up for negotiation.